### ISSUE TITLE
Log-Eintrag mit js-Controller 3.3.x

### ISSUE THREAD

**Issue by e-i-k-e** [phifogg/ioBroker.sainlogic#74]
Nach Update des js-Controllers wird nachfolgendes im Log angezeigt.

```

sainlogic.0 | 2021-04-30 21:29:37.154 | warn | (6166) State value to set for "sainlogic.0.weather.current.uvi" has wrong type "string" but has to be "number"
-- | -- | -- | --
sainlogic.0 | 2021-04-30 21:29:37.154 | warn | (6166) State value to set for "sainlogic.0.weather.current.winddir" has wrong type "string" but has to be "number"
sainlogic.0 | 2021-04-30 21:29:37.153 | warn | (6166) State value to set for "sainlogic.0.weather.current.outdoorhumidity" has wrong type "string" but has to be "number"
sainlogic.0 | 2021-04-30 21:29:37.153 | warn | (6166) State value to set for "sainlogic.0.weather.current.indoorhumidity" has wrong type "string" but has to be "number"
sainlogic.0 | 2021-04-30 21:29:37.141 | warn | (6166) State "sainlogic.0.info.last_listener_update" has no existing object, this might lead to an error in future versions
sainlogic.0 | 2021-04-30 19:28:36.125 | warn | (6166) State value to set for "sainlogic.0.weather.current.uvi" has wrong type "string" but has to be "number"
sainlogic.0 | 2021-04-30 19:28:36.124 | warn | (6166) State value to set for "sainlogic.0.weather.current.winddir" has wrong type "string" but has to be "number"
sainlogic.0 | 2021-04-30 19:28:36.124 | warn | (6166) State value to set for "sainlogic.0.weather.current.outdoorhumidity" has wrong type "string" but has to be "number"
sainlogic.0 | 2021-04-30 19:28:36.123 | warn | (6166) State value to set for "sainlogic.0.weather.current.indoorhumidity" has wrong type "string" but has to be "number"
sainlogic.0 | 2021-04-30 19:28:36.121 | warn | (6166) State "sainlogic.0.info.last_listener_update" has no existing object, this might lead to an error in future versions
sainlogic.0 | 2021-04-30 19:27:35.128 | warn | (6166) State value to set for "sainlogic.0.weather.current.uvi" has wrong type "string" but has to be "number"
sainlogic.0 | 2021-04-30 19:27:35.128 | warn | (6166) State value to set for "sainlogic.0.weather.current.winddir" has wrong type "string" but has to be "number"
sainlogic.0 | 2021-04-30 19:27:35.127 | warn | (6166) State value to set for "sainlogic.0.weather.current.outdoorhumidity" has wrong type "string" but has to be "number"
sainlogic.0 | 2021-04-30 19:27:35.126 | warn | (6166) State value to set for "sainlogic.0.weather.current.indoorhumidity" has wrong type "string" but has to be "number"
sainlogic.0 | 2021-04-30 19:27:35.122 | warn | (6166) State "sainlogic.0.info.last_listener_update" has no existing object, this might lead to an error in future versions
sainlogic.0 | 2021-04-30 19:26:34.171 | warn | (6166) State value to set for "sainlogic.0.weather.current.uvi" has wrong type "string" but has to be "number"
sainlogic.0 | 2021-04-30 19:26:34.170 | warn | (6166) State value to set for "sainlogic.0.weather.current.winddir" has wrong type "string" but has to be "number"
sainlogic.0 | 2021-04-30 19:26:34.170 | warn | (6166) State value to set for "sainlogic.0.weather.current.outdoorhumidity" has wrong type "string" but has to be "number"
sainlogic.0 | 2021-04-30 19:26:34.170 | warn | (6166) State value to set for "sainlogic.0.weather.current.indoorhumidity" has wrong type "string" but has to be "number"
sainlogic.0 | 2021-04-30 19:26:34.167 | warn | (6166) State "sainlogic.0.info.last_listener_update" has no existing object, this might lead to an error in future versions


```

**Comment by HerbertJoos** [comment]
Auch bei mir, nach Update des js-Controllers wird nachfolgendes im Log angezeigt.

`2021-05-01 12:02:27.406 - warn: deconz.0 (2093) State value to set for "deconz.0.lights.ccccccfffeade53d.bri" has wrong type "undefined" but has to be "number"
2021-05-01 12:02:27.571 - warn: deconz.0 (2093) State value to set for "deconz.0.lights.ccccccfffeade53d.xy" has wrong type "object" but has to be "string"
2021-05-01 12:02:27.574 - warn: deconz.0 (2093) State value to set for "deconz.0.lights.ccccccfffeade53d.xy" has wrong type "object" but has to be "string"
2021-05-01 12:02:27.664 - warn: deconz.0 (2093) State value to set for "deconz.0.lights.ccccccfffeade53d.xy" has wrong type "object" but has to be "string"
2021-05-01 12:02:27.712 - warn: deconz.0 (2093) State value to set for "deconz.0.lights.ccccccfffeade53d.xy" has wrong type "object" but has to be "string"
2021-05-01 12:02:28.407 - warn: deconz.0 (2093) State value to set for "deconz.0.lights.ccccccfffeade53d.xy" has wrong type "undefined" but has to be "string"
2021-05-01 12:02:28.659 - warn: deconz.0 (2093) State value to set for "deconz.0.lights.ccccccfffeade53d.xy" has wrong type "object" but has to be "string"
2021-05-01 12:02:28.700 - warn: deconz.0 (2093) State value to set for "deconz.0.lights.ccccccfffeade53d.xy" has wrong type "object" but has to be "string"
2021-05-01 12:03:27.022 - warn: telegram.0 (3466) Read-only state "telegram.0.communicate.botSendMessageId" has been written without ack-flag with value "124847"
2021-05-01 12:03:27.023 - warn: telegram.0 (3466) State value to set for "telegram.0.communicate.botSendMessageId" has wrong type "number" but has to be "string"
2021-05-01 12:03:27.027 - warn: telegram.0 (3466) Read-only state "telegram.0.communicate.botSendChatId" has been written without ack-flag with value "803664695"
2021-05-01 12:03:27.028 - warn: telegram.0 (3466) State value to set for "telegram.0.communicate.botSendChatId" has wrong type "number" but has to be "string"
2021-05-01 12:04:49.519 - warn: telegram.0 (3466) Read-only state "telegram.0.communicate.botSendMessageId" has been written without ack-flag with value "124848"
2021-05-01 12:04:49.520 - warn: telegram.0 (3466) State value to set for "telegram.0.communicate.botSendMessageId" has wrong type "number" but has to be "string"
2021-05-01 12:04:49.524 - warn: telegram.0 (3466) Read-only state "telegram.0.communicate.botSendChatId" has been written without ack-flag with value "803664695"
2021-05-01 12:04:49.524 - warn: telegram.0 (3466) State value to set for "telegram.0.communicate.botSendChatId" has wrong type "number" but has to be "string"
2021-05-01 12:10:39.242 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_099950226513.windowopenactiveendtime" has wrong type "object" but has to be "number"
2021-05-01 12:10:39.248 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_099950226513.boostactiveendtime" has wrong type "object" but has to be "number"
2021-05-01 12:10:39.283 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_099950226513.endperiod" has wrong type "object" but has to be "number"
2021-05-01 12:10:39.331 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_099950204185.windowopenactiveendtime" has wrong type "object" but has to be "number"
2021-05-01 12:10:39.332 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_099950204185.boostactiveendtime" has wrong type "object" but has to be "number"
2021-05-01 12:10:39.340 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_099950204185.endperiod" has wrong type "object" but has to be "number"
2021-05-01 12:10:39.408 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_099950280184.windowopenactiveendtime" has wrong type "object" but has to be "number"
2021-05-01 12:10:39.409 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_099950280184.boostactiveendtime" has wrong type "object" but has to be "number"
2021-05-01 12:10:39.419 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_099950280184.endperiod" has wrong type "object" but has to be "number"
2021-05-01 12:10:39.491 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_099950154897.windowopenactiveendtime" has wrong type "object" but has to be "number"
2021-05-01 12:10:39.492 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_099950154897.boostactiveendtime" has wrong type "object" but has to be "number"
2021-05-01 12:10:39.494 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_099950154897.endperiod" has wrong type "object" but has to be "number"
2021-05-01 12:10:39.561 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_099950194682.windowopenactiveendtime" has wrong type "object" but has to be "number"
2021-05-01 12:10:39.562 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_099950194682.boostactiveendtime" has wrong type "object" but has to be "number"
2021-05-01 12:10:39.563 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_099950194682.endperiod" has wrong type "object" but has to be "number"
2021-05-01 12:10:39.661 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_099950206774.windowopenactiveendtime" has wrong type "object" but has to be "number"
2021-05-01 12:10:39.666 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_099950206774.boostactiveendtime" has wrong type "object" but has to be "number"
2021-05-01 12:10:39.667 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_099950206774.endperiod" has wrong type "object" but has to be "number"
2021-05-01 12:10:40.199 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_E8:FC:56-900.windowopenactiveendtime" has wrong type "object" but has to be "number"
2021-05-01 12:10:40.200 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_E8:FC:56-900.boostactiveendtime" has wrong type "object" but has to be "number"
2021-05-01 12:10:40.201 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_E8:FC:56-900.endperiod" has wrong type "object" but has to be "number"`

**Comment by HerbertJoos** [comment]
Auch bei mir, nach Update des js-Controllers wird nachfolgendes im Log angezeigt.

`2021-05-01` 12:02:27.406 - warn: deconz.0 (2093) State value to set for "deconz.0.lights.ccccccfffeade53d.bri" has wrong type "undefined" but has to be "number"
2021-05-01 12:02:27.571 - warn: deconz.0 (2093) State value to set for "deconz.0.lights.ccccccfffeade53d.xy" has wrong type "object" but has to be "string"
2021-05-01 12:02:27.574 - warn: deconz.0 (2093) State value to set for "deconz.0.lights.ccccccfffeade53d.xy" has wrong type "object" but has to be "string"
2021-05-01 12:02:27.664 - warn: deconz.0 (2093) State value to set for "deconz.0.lights.ccccccfffeade53d.xy" has wrong type "object" but has to be "string"
2021-05-01 12:02:27.712 - warn: deconz.0 (2093) State value to set for "deconz.0.lights.ccccccfffeade53d.xy" has wrong type "object" but has to be "string"
2021-05-01 12:02:28.407 - warn: deconz.0 (2093) State value to set for "deconz.0.lights.ccccccfffeade53d.xy" has wrong type "undefined" but has to be "string"
2021-05-01 12:02:28.659 - warn: deconz.0 (2093) State value to set for "deconz.0.lights.ccccccfffeade53d.xy" has wrong type "object" but has to be "string"
2021-05-01 12:02:28.700 - warn: deconz.0 (2093) State value to set for "deconz.0.lights.ccccccfffeade53d.xy" has wrong type "object" but has to be "string"
2021-05-01 12:03:27.022 - warn: telegram.0 (3466) Read-only state "telegram.0.communicate.botSendMessageId" has been written without ack-flag with value "124847"
2021-05-01 12:03:27.023 - warn: telegram.0 (3466) State value to set for "telegram.0.communicate.botSendMessageId" has wrong type "number" but has to be "string"
2021-05-01 12:03:27.027 - warn: telegram.0 (3466) Read-only state "telegram.0.communicate.botSendChatId" has been written without ack-flag with value "803664695"
2021-05-01 12:03:27.028 - warn: telegram.0 (3466) State value to set for "telegram.0.communicate.botSendChatId" has wrong type "number" but has to be "string"
2021-05-01 12:04:49.519 - warn: telegram.0 (3466) Read-only state "telegram.0.communicate.botSendMessageId" has been written without ack-flag with value "124848"
2021-05-01 12:04:49.520 - warn: telegram.0 (3466) State value to set for "telegram.0.communicate.botSendMessageId" has wrong type "number" but has to be "string"
2021-05-01 12:04:49.524 - warn: telegram.0 (3466) Read-only state "telegram.0.communicate.botSendChatId" has been written without ack-flag with value "803664695"
2021-05-01 12:04:49.524 - warn: telegram.0 (3466) State value to set for "telegram.0.communicate.botSendChatId" has wrong type "number" but has to be "string"
2021-05-01 12:10:39.242 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_099950226513.windowopenactiveendtime" has wrong type "object" but has to be "number"
2021-05-01 12:10:39.248 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_099950226513.boostactiveendtime" has wrong type "object" but has to be "number"
2021-05-01 12:10:39.283 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_099950226513.endperiod" has wrong type "object" but has to be "number"
2021-05-01 12:10:39.331 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_099950204185.windowopenactiveendtime" has wrong type "object" but has to be "number"
2021-05-01 12:10:39.332 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_099950204185.boostactiveendtime" has wrong type "object" but has to be "number"
2021-05-01 12:10:39.340 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_099950204185.endperiod" has wrong type "object" but has to be "number"
2021-05-01 12:10:39.408 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_099950280184.windowopenactiveendtime" has wrong type "object" but has to be "number"
2021-05-01 12:10:39.409 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_099950280184.boostactiveendtime" has wrong type "object" but has to be "number"
2021-05-01 12:10:39.419 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_099950280184.endperiod" has wrong type "object" but has to be "number"
2021-05-01 12:10:39.491 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_099950154897.windowopenactiveendtime" has wrong type "object" but has to be "number"
2021-05-01 12:10:39.492 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_099950154897.boostactiveendtime" has wrong type "object" but has to be "number"
2021-05-01 12:10:39.494 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_099950154897.endperiod" has wrong type "object" but has to be "number"
2021-05-01 12:10:39.561 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_099950194682.windowopenactiveendtime" has wrong type "object" but has to be "number"
2021-05-01 12:10:39.562 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_099950194682.boostactiveendtime" has wrong type "object" but has to be "number"
2021-05-01 12:10:39.563 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_099950194682.endperiod" has wrong type "object" but has to be "number"
2021-05-01 12:10:39.661 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_099950206774.windowopenactiveendtime" has wrong type "object" but has to be "number"
2021-05-01 12:10:39.666 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_099950206774.boostactiveendtime" has wrong type "object" but has to be "number"
2021-05-01 12:10:39.667 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_099950206774.endperiod" has wrong type "object" but has to be "number"
2021-05-01 12:10:40.199 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_E8:FC:56-900.windowopenactiveendtime" has wrong type "object" but has to be "number"
2021-05-01 12:10:40.200 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_E8:FC:56-900.boostactiveendtime" has wrong type "object" but has to be "number"
2021-05-01 12:10:40.201 - warn: fritzdect.0 (13627) State value to set for "fritzdect.0.DECT_E8:FC:56-900.endperiod" has wrong type "object" but has to be "number"

**Comment by phifogg** [comment]
Ich kriege gerade die neue jsController version bei mir nicht installiert.. das muss ich erst noch rausknobeln wie ..

Wenn Du aber testen magst habe ich im DEV Branch versucht die Warn-Meldungen zu umgehen.
https://github.com/phifogg/ioBroker.sainlogic/tree/dev

**Comment by phifogg** [comment]
Fixed with #79
